File: 백준/Bronze/25304. 영수증/영수증.py
total_price = int(input())
total_count = int(input())

sum_price = 0

for i in range(1, total_count + 1):
    price, count = map(int, input().split())
    sum_price = sum_price + (price * count)

if sum_price == total_price:
    print("Yes")
else:
    print("No")

File: 백준/Bronze/25304. 영수증/영수증.js
const fs = require("fs");
const input = fs.readFileSync(0, "utf-8").trim().split("\n");

const totalPrice = Number(input[0]);
const n = Number(input[1]) + 1;
let myPrice = 0;

for (let i = 2; i <= n; i++) {
  let [price, count] = input[i].split(" ");
  myPrice += Number(price) * Number(count);
}

console.log(totalPrice === myPrice ? "Yes" : "No");
